bluemira/equilibria/coils/_coil.py

```diff
@@ -65,6 +65,13 @@ class CoilType(Enum, metaclass=CoilTypeEnumMeta):
     DUM = auto()
     NONE = auto()
 
+    @classmethod
+    def get_actively_controlled_types(cls):
+        """Get the list of types which are actively controlled
+        (as opposed to passive coils).
+        """
+        return [cls.PF, cls.CS]
+
```

Comment:
I know where you're coming from but I don't think this is a good idea. This implicitly restricts the user and will only be used in eqdsk read in. 

Remove please

Reply:
Yep, understood, and will delete.